Source/WebKit/Shared/WebsiteData/WebsiteData.cpp

```diff
@@ -75,7 +75,7 @@ WebsiteDataProcessType WebsiteData::ownerProcess(WebsiteDataType dataType)
         return WebsiteDataProcessType::UI;
     case WebsiteDataType::PrivateClickMeasurements:
         return WebsiteDataProcessType::Network;
-#if HAVE(CFNETWORK_ALTERNATIVE_SERVICE)
+#if HAVE(CFNETWORK_ALTERNATIVE_SERVICE) || USE(CURL)
```

Comment:
Maybe we should introduce a HAVE(ALTERNATIVE_SERVICE) macro that is true if we have CFNetwork alt-svc support or cURL.  Fine to do in a separate PR.

Reply:
OK, I will fix it in a separate PR.

Reply:
Fixed in this PR.